util/test/perf/perfgraph.js

```diff
@@ -732,7 +778,8 @@ function customValueFormatter(val, opts, series_name, dygraph) {
 
   // update digits, but do NOT redraw. Then use the default value formatter
   dygraph.updateOptions({digitsAfterDecimal: digits}, true);
-  return Dygraph.numberValueFormatter(val, opts);
+
+  return val.toFixed(digits);
```

Comment:
This effectively means we lose the `labelsKMB` and `labelsKMB2` options, right? I don't think we use them now so that should be ok.

Does this also mean we'll lose the auto-scientific notation switch?